src/AppBundle/Form/SiteType.php

```diff
@@ -61,6 +65,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     'help-block' => 'Domain names associated with this site'
                 ]
             ])
+            ->add('author', null, ['required' => true])
```

Comment:
Delete this

Create a Doctrine Listener

onPerset  - setAuthor
onUpdate setEditor

to the logged in user